README.md

```diff
@@ -44,6 +44,8 @@ The following attributes are set within *config.json*.
 
 >**body** (string): Introductory or explanatory text. For example, "Welcome to ACME Learning. This course is available in several languages. Please make a selection. "
 
+>**_showOnCourseLoad** (boolean): Determines whether the language picker will be displayed on course load. If set to false, the course will load with the default language selected.
```

Comment:
could you add ' and the user will only be able to change languages by clicking the icon in the navigation bar.' to that last sentence?